share/store/file.go

```diff
@@ -0,0 +1,21 @@
+package store
+
+import (
+	"context"
+	"io"
+
+	"github.com/celestiaorg/nmt"
+	"github.com/celestiaorg/rsmt2d"
+
+	"github.com/celestiaorg/celestia-node/share"
+)
+
+type File interface {
```

Comment:
Please add comments for the File. In general, treat everything we do in the protocol_v2 branch as we do it in main